examples/framejs-nextjs-display-info/tailwind.config.ts

```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
I dont see the usage of pages or components, I wonder if we should keep those lines or not?